packages/chat-core/src/index.ts

```diff
@@ -157,7 +157,7 @@ export interface ConversationHandler {
   sendWelcomeIntent: (context?: Context) => void;
   sendIntent: (intentId: string, context?: Context) => void;
   sendStructured: (request: StructuredRequest, context?: Context) => void;
-  subscribe: (subscriber: Subscriber) => void;
+  subscribe: (subscriber: Subscriber) => () => void;
```

Comment:
@michaelglass I don't understand how this is the correct type, the subscriber is already a function.

Reply:
the subscribe function returns an unsubscribe function for that subscriber

Reply:
You're so right, thanks for straightening that out!